### PR TITLE
Updated some records

### PR DESCRIPTION
**Pull request by mal-tee**
🤖 👀

### CODE REVIEW

File: companies/3dhubs.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "name": "3D Hubs B.V.",
     "address": "Frederiksplein 42\n1017 XN Amsterdam\nNetherlands",
     "phone": "+31 20 261 1900",
```
Comment (baltpeter) [inline companies/3dhubs.json:11]: They seem to have also changed their address and phone number. Would you mind updating those as well, while you're at it? :)

**Review by baltpeter — APPROVED** [review]
Very cool!

File: companies/3dhubs.json (hunk 1 above)
Comment (mal-tee) [reply companies/3dhubs.json]: Sure.

**Review by baltpeter — APPROVED** [review]
Awesome, thanks!

**Pull request merged**